src/components/post/index.js

```diff
@@ -36,20 +41,40 @@ const Post = ({ name, date, content, comments = [], likes = 0 }) => {
                 </section>
 
                 <section className={`post-interactions-container border-top ${comments.length ? 'border-bottom' : null}`}>
-                    <div className="post-interactions">
-                        <div>Like</div>
-                        <div>Comment</div>
+                    <div id="likeButton" className="post-interactions">
+                        {/* make appropriate click events: linked to liked API */}
+                        <div className="onHover">
+                            <button className={isLiked === true ? true : false}
+                                onClick={() => {
+                                    console.log("clicked Like!")
```

Comment:
Let's remove all console logs, you don't want debug statements in the main branch